back-end/API/app/services/auth_service.py

```
@@ -0,0 +1,74 @@
+        dto.password = hasher.hash(dto.password)
+        dto.enterpriseId = None
+        dto.enterprise = None
+        dto.isModerator = False
+        dto.verified = False
+        dto.active = True
+
+        new_user = user_repo.create(dto)
+        token = self._generateToken(new_user)
+        return token, new_user
+
+    def _generateToken(self, user):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_generateToken`

(ANN202)